ch3/ex03.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+# EX 03: create a new list with 1 integer and 2 strings as elements
+
+import random
+
+def main():
+	lista = []
+	lista.append(random.randint(1,100))
+	for _ in xrange(2):
+		lista.append(random.choice('abcdefghijklmnopqrstuvwxyz'))
```

Comment:
It's true that a single character is a string in python, when I read random string I'd expect to see a sequence of characters randomly chosen (and of random length maybe?)